src/hotspot/share/c1/c1_LinearScan.hpp

```diff
@@ -634,7 +634,9 @@ class Interval : public CompilationResourceObj {
   // printing
 #ifndef PRODUCT
   void print() const { print_on(tty); }
-  void print_on(outputStream* out) const;
+  void print_on(outputStream* out) const {
+    print_on(out, false);
+  }
   // Special version for compatibility with C1 Visualizer.
   void print_on(outputStream* out, bool is_cfg_printer) const;
```

Comment:
then why not just print_on(outputStream* out, bool is_cfg_printer=false) const directly? those 3 lines can be further saved. :)

Reply:
This would be a good idea but unfortunately that does not compile as it complains that `virtual void AllocatedObj::print_on(outputStream*) const` was hidden.

Reply:
you are right.  I just figure out that print_on(outputStream*) const is virtual.  we can't use that trick.  
anyway, your patch looks good to me.